pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>de.studiocode</groupId>
     <artifactId>MiniatureBlocks</artifactId>
-    <version>0.1-SNAPSHOT</version>
+    <version>0.3-SNAPSHOT</version>
```

Comment:
shouldn't it be 0.3 without snapshot since 0.3 has been published already?

Reply:
And from now on it should also be 0.4-SNAPSHOT because that will be the next version!